modules/core/shared/src/main/scala/lucuma/core/model/ConstraintSet.scala

```diff
@@ -17,8 +15,8 @@ import monocle.Optional
 
 /** Constraints for an observation. */
 case class ConstraintSet(
-  imageQuality:    ImageQuality,
-  cloudExtinction: CloudExtinction,
+  imageQuality:    ImageQuality.Point,
+  cloudExtinction: CloudExtinction.Point,
```

Comment:
I wonder if something like `Constraint` would make sense if we only plan on using the enums for the constraint set. Then it would read `ImageQuality.Constraint.PointOne` instead of `ImageQuality.Point.PointOne`.

Reply:
I've struggled with this name and I couldn't find anything that conveys the meaning of a "singled-out instance".

`Constraint` seems to describe more our usage than the general "singled-out instance".

Other names I've considered, in case you think any of them are a better fit: `Stop`, `Instance`, `Mark`, `Peg`, `Hint`. IMHO, none of those looked too good in an API.

Reply:
`Preset` maybe?  Idk, it's fine however if you want to leave it as is.

Reply:
I like `Preset`, let's go with that one.